src/main/java/fact/TriggerEmulation/Discriminator.java

```diff
@@ -0,0 +1,120 @@
+package fact.TriggerEmulation;
+
+import fact.Constants;
+
+/**
+ *Class with functions to discriminate the signals of a given patch (or a bunch of them).
+ * Furhtermore it contains helper functions for the discrimination e.g. conversion between DAC and mV
+ */
+public class Discriminator {
+
+    public static int default_slice = 0;
+
+    /**
+     *Discriminate the signal of a given patch
+     * @param data timeseries, array[n_pixels_pe_patch]
```

Comment:
I'm sorry .. didn't read java for some time .. by `array[n_pixels_pe_patch]` you mean: 

This is an array and the length is equal to the number of pixels we have in a patch? Which is 9, right?
In the implementation this looks like a it is a timeseries with a length of maybe 300 or so.... 
confused.

Reply:
nice catch, that was a copy&paste failure. Yes, you are right. It's the lenght is number of time slices of the series. Since this more confusing than helping I delete this.